ci-operator/step-registry/ipi/install/install/ipi-install-install-commands.sh

```diff
@@ -348,8 +348,33 @@ date "+%F %X" > "${SHARED_DIR}/CLUSTER_INSTALL_END_TIME"
 
 if test "${ret}" -eq 0 ; then
   touch  "${SHARED_DIR}/success"
-  # Save console URL in `console.url` file so that ci-chat-bot could report success
-  echo "https://$(env KUBECONFIG=${dir}/auth/kubeconfig oc -n openshift-console get routes console -o=jsonpath='{.spec.host}')" > "${SHARED_DIR}/console.url"
+
+  CONSOLE_ENABLED=false
+  CLUSTER_VERSION="$(oc --kubeconfig "${dir}/auth/kubeconfig" get -o json clusterversion version)"
+  if [ -z "${CLUSTER_VERSION}" ]; then
+    echo "Unabled to retrieve ClusterVersion to check enabledCapabilities"
+  else
+    KNOWN_CAPABILITIES="$(printf '%s' "${CLUSTER_VERSION}" | jq '.status.capabilities.knownCapabilities')"
+    ENABLED_CAPABILITIES="$(printf '%s' "${CLUSTER_VERSION}" | jq '.status.capabilities.enabledCapabilities')"
```

Comment:
[`/bin/bash: line 358: jq: command not found`][1], so we should either find an alternative approach to processing the ClusterVersion contents, find a way to install `jq`, or maybe process the installer output instead of trying to check in ClusterVersion...

https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/32262/rehearse-32262-pull-ci-openshift-cluster-machine-approver-master-e2e-aws-capi-techpreview/1569923695856586752#1:build-log.txt%3A81